tests/dummy/app/templates/signup-example.hbs

```diff
@@ -1,20 +1,20 @@
 <h1>Signup example</h1>
 
-<form {{action "signup" on="submit"}}>
+<form class="signup-example-form" {{action "signup" on="submit"}}>
```

Comment:
I'm referring to the dummy/app/component that we could also use for both testing in integration and acceptance.

- https://github.com/cibernox/ember-native-dom-helpers/blob/master/tests/dummy/app/templates/components/x-form.hbs
- https://github.com/cibernox/ember-native-dom-helpers/blob/master/tests/dummy/app/components/x-form.js